src/main/g8/src/main/scala/$package$/Main.scala

```diff
@@ -10,8 +13,20 @@ import akka.management.scaladsl.AkkaManagement
 
 object Main {
 
+  val logger = LoggerFactory.getLogger(getClass)
+  
   def main(args: Array[String]): Unit = {
-    ActorSystem[Nothing](Main(), "$name;format="word-space,upper-camel"$")
+
+    val serviceName = "$name;format="word-space,upper-camel"$"
+    
+    if (args.nonEmpty) {
+      // if an argument is passed, it's expected to be an alternative config file
+      val configToLoad = args(0)
+      logger.info(s"Actor system will be initialized with configuration file [\$configToLoad]")
+      ActorSystem[Nothing](Main(), serviceName, ConfigFactory.load(configToLoad))
+    } else {
+      ActorSystem[Nothing](Main(), serviceName)
+    }
```

Comment:
The drawback is that we are making it slightly more complex (`if (devMode)`).
I see the advantage and don't have a strong opinion, but it's not an immediate approval for me. Let's see what other's think.